pkg/core/config/main.go

```diff
@@ -172,7 +172,7 @@ func (config *Config) Validate() error {
 		// Try to guess SourceID
 		if len(t.SourceID) == 0 && len(config.Sources) > 1 {
 
-			logrus.Errorf("{empty 'sourceID' for target '%s'", id)
+			logrus.Errorf("empty 'sourceID' for target '%s'", id)
```

Comment:
```suggestion
			logrus.Errorf("empty 'sourceID' for target %q", id)
```

I find it clearer to use %q instead of '%s'